andy-db.txt

```diff
@@ -0,0 +1,41 @@
+Database & Common Used Queries 
+    • What is Database
+    //answer
+    Database is a collection of data based on certain conditions that are interconnected so that it is easy to manage it. and also from the management it can be easy to find, store and dispose of information
+
+    • NoSQL vs SQL
```

Comment:
can you please explain about what is SQL and NoSQL ? Why and when we use SQL or NoSQL databases ? Thank you